### PR TITLE
deps update + prep for release

### PR DESCRIPTION
**Pull request by lindsey-w**
### Background

Update dependencies + prep for release

### Changes

* update deps
* prep for next `pat` release

### Testing

* `make test`
* manual testing:
* `python setup.py sdist` -> pip3 install -> `pat upload`

### CODE REVIEW

**Comment by danbiwer** [comment]
I can build this successfully and run all commands, but the circle ci build step is failing.  The module 'typing_extensions' isn't referenced anywhere in this repo, and I can run the failing ci command locally with no errors.

Any ideas here?  @michael-cariello @cdzombak @lindsey-w

**Comment by danbiwer** [comment]
Hijacking this PR:

This fixes setup.py so that we don't have missing dependencies

**Review by danbiwer — DISMISSED** [review]

**Review by darwayne — APPROVED** [review]

File: Pipfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ bandit = "==1.7.0"
 black = "==20.8b1"
 isort = "==5.8.0"
 mypy = "==0.812"
-panther-core = "==0.0.9"
+panther-core = "==0.0.10"
```
Comment (darwayne) [inline Pipfile:11]: is this right? shouldn't this be 0.1.0?

**Review by darwayne — CHANGES_REQUESTED** [review]
yeah we should make sure version is right: https://pypi.org/project/panther-core/0.2.0/#history

**Comment by darwayne** [comment]
P.S ... just added a 0.2.0 for panther_core so we could go with that .. should have non breaking changes

**Comment by darwayne** [comment]
not sure if 0.0.9 -> 0.1.0 had breaking changes though

File: Pipfile (hunk 1 above)
Comment (danbiwer) [reply Pipfile:11]: currently the detections engine/executor are using 0.0.10.  Is that going to change in this upcoming release?

Comment (danbiwer) [reply Pipfile:11]: from release-1.40's requirements.txt
```
panther-core==0.0.10
```

Comment (darwayne) [reply Pipfile:11]: ahh I missed that we actually went up to 0.0.10 in the pip .. my bad ... yeah we have some newer versions coming that should probably go out in a separate PR

**Review by darwayne — APPROVED** [review]

**Comment by darwayne** [comment]
Ok I was wrong, didn't realize we had a 0.0.10 version .. saw 0.1.0 and thought this was wrong

**Review by nmartin413 — APPROVED** [review]

**Pull request merged**